LICENSE

```diff
@@ -1,4 +1,8 @@
-Copyright (c) 2013-2015, Alfred Klomp
+Copyright (c) 2005-2007, Nick Galbreath
+Copyright (c) 2013-2017, Alfred Klomp
+Copyright (c) 2015-2017, Wojciech Mula
+Copyright (c) 2016-2017, Matthieu Darbois
```

Comment:
The code from Nick Galbreath has been released under the BSD license and is used in chromium for example. Since it's BSD, it can be included as long as a copyright is maintained.
I mentioned him in the commit & PR so that he knows his code has been used here.